packages/IMAPClient-UI.package/ICFolderDialog.class/instance/editAccount.with..st

```diff
@@ -1,7 +1,7 @@
 accountManagment
 editAccount: anAccountInfo with: anEndPoint
 	
-	ICEndPoint removeConfiguration: anEndPoint.
+	ICEndPoint removeConfiguration: anEndPoint accountInfo asDictionary.
```

Comment:
I made a small change in a commit so the method naming is similar to deleteAccountButton